### PR TITLE
[React | Frontend]Refactor Frontend React

### PR DESCRIPTION
**Pull request by dcarrilloWolox**
## Summary

- Moved `/react`  inside `/frontend` & deleted 'React' from the main page. 
- Extracted `/react-native` from `/react` and added a link to `/react-native` in the main page.
- Moved vscode style guide to `/front-end#text-editors`.
- Deleted de link and the sources about react rails.
- Merged `/react/README.md` and `/react/react/README.md`

### Main page
selection_199

### Frontend main
selection_200

### React Main
selection_201

### Internal Libraries
selection_202

## Trello card
https://trello.com/c/OZDoZHLM/4-erradicar-react-in-rails-de-la-faz-de-la-tierra-y-sacar-rn-de-react

### CODE REVIEW

File: frontend/react/README.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-## Open Source
-
-- [React Chat Widget](https://github.com/Wolox/react-chat-widget)
-
-- [React App Rewire Wolox](https://github.com/Wolox/react-app-rewire-wolox)
-
 ## Recommended Libraries
+
 - [Internal Libraries](./docs/libraries/internal-libraries.md)
+
```
Comment (FrankIglesias) [inline frontend/react/README.md]: delete this line

File: frontend/react/README.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 - [External Libraries](./docs/libraries/external-libraries.md)
 
+## Useful Documentation
+
+- [Style Guide](./style-guide.md)
+
```
Comment (FrankIglesias) [inline frontend/react/README.md]: same

File: react-native/README.md [hunk 3]
```diff
@@ -39,6 +39,7 @@ Download it [here.](./docs/atom-snippets.cson)
 
 ## Posts
 
+- [Introducing redux-recompose](https://medium.com/wolox-driving-innovation/932e746b0198)
```
Comment (FrankIglesias) [inline react-native/README.md]: https://medium.com/wolox-driving-innovation/easy-data-management-from-a-rest-api-using-redux-recompose-v2-7c4dc5323445
https://medium.com/wolox-driving-innovation/fighting-back-fire-with-firebase-dc3e47021a81
https://medium.com/wolox-driving-innovation/the-importance-of-separating-structure-from-style-fa349e230b51

File: frontend/react/README.md (hunk 1 above)
Comment (FrankIglesias) [inline frontend/react/README.md]: delete this line

File: react-native/README.md (hunk 3 above)
Comment (SKOLZ) [reply react-native/README.md]: the last one shouldn't be added since RN doesn't use html and css

File: frontend/README.md [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 - [AngularJS](./angular-js/)
 
-- [React](../react/)
+- [React](./react/README.md)
```
Comment (SKOLZ) [inline frontend/README.md]: I think you don't need the /README.md, just keep the:
```
- [React](./react/)
```

like the rest of the techs

**Pull request merged**